packages/jellyfish-api-core/__tests__/category/account/removePoolLiquidity.test.ts

```diff
@@ -0,0 +1,112 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  let poolLiquidityAddress: string
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(200)
+
+    await createToken('DDAI')
+    await mintTokens('DDAI')
+    await createPoolPair('DDAI')
+    await addPoolLiquidity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  async function createToken (symbol: string): Promise<void> {
+    const address = await container.call('getnewaddress')
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+  }
+
+  async function createPoolPair (tokenB: string, metadata?: any): Promise<void> {
```

Comment:
Do I remove the following? Since these utilities are active in `@defichain/testing`. 

 ```
async function createToken {}...
 async function mintTokens {}...
 async function createPoolPair {}...
 async function addPoolLiquidity ()...
```

Reply:
yea those can be removed ady